### ISSUE TITLE
Ctrl-Z doesn't suspend doitlive

### ISSUE THREAD

**Issue by emanuelhouse** [sloria/doitlive#44]
I'd like to be able to pause/resume doitlive at arbitrary points during my presentation in case I happen to field a question that would best be answered or supplemented by an ad-hoc invocation of command. I was simply thinking that using Ctrl-Z to suspend doitlive and later `fg` to resume it would be the best way to get there, but Ctrl-Z appears to be treated as a regular keypress.

I'm using the Homebrew version of doitlive
```
> doitlive --version
doitlive, version 4.0.0
> uname -a
Darwin ehouse-mbp 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64 x86_64
> which doitlive
/usr/local/bin/doitlive
```

**Comment by sloria** [comment]
Thanks for reporting this @emanuelhouse . This is now implemented. I'll release it soon.

**Comment by sloria** [comment]
This is released in 4.0.1
